api/clients/v2/payloadretrieval/relay_payload_retriever_test.go

```diff
@@ -35,18 +35,18 @@ const maxPayloadBytes = 1025 // arbitrary value
 
 type RelayPayloadRetrieverTester struct {
 	Random                *testrandom.TestRandom
-	RelayPayloadRetriever *clients.RelayPayloadRetriever
+	RelayPayloadRetriever *RelayPayloadRetriever
 	MockRelayClient       *clientsmock.MockRelayClient
 	G1Srs                 []bn254.G1Affine
-	Config                clients.RelayPayloadRetrieverConfig
+	Config                RelayPayloadRetrieverConfig
```

Comment:
Is this config used for anything? Or is it just passed to the RelayPayloadRetriever constructor? Why keep it around?

Reply:
> Guessing buildBlobAndCert is a test function you use for testing only and need this config?

Correct. To make this less confusing, I've removed the config from the test harness, and simply put the single piece of info there that is actually needed by the test

[37c48c56](https://github.com/Layr-Labs/eigenda/pull/1368/commits/37c48c564e76b0b74c0442ace759efac024caf1c)